ProjectSettings/QualitySettings.asset

```diff
@@ -215,14 +283,74 @@ QualitySettings:
     asyncUploadBufferSize: 16
     asyncUploadPersistentBuffer: 1
     resolutionScalingFixedDPIFactor: 1
+    customRenderPipeline: {fileID: 0}
```

Comment:
Oh. A lot of settings changed/added...
Is it really needed? Just challenging. ;-)

Reply:
This only looks like this, because I have created an extra quality level for the Android devices.

![image](https://user-images.githubusercontent.com/12483468/234043460-b09879cc-de4f-4fe3-a306-1fab84ffeecd.png)